flocker/provision/_aws.py

```diff
@@ -51,6 +53,32 @@
 INSTANCE_TIMEOUT = 300
 
 
+class EliotLogHandler(logging.Handler):
+    # Whitelist ``"msg": "Params:`` field for logging.
+    _to_log = {"Params"}
+
+    def emit(self, record):
+        fields = vars(record)
```

Comment:
Why not just access the `record.msg` attribute? 
 * https://docs.python.org/2/library/logging.html#logging.LogRecord